content/using/install.md

```diff
@@ -6,37 +6,33 @@ template = "page_indiced.html"
 aliases = ["/docs/getting-started/"]
 +++
 
-This guide covers how to set up and install your own Urbit. Running Urbit on your own is somewhat technical, so if you're not technically inclined you may want to consider a [hosting provider](#hosting-providers).
+This guide covers how to install and set up your own planet, but installation for a star is exactly the same. You will first install the Urbit binary, then you will either boot a comet or a planet.  It's not too difficult, but if you would prefer to have someone run your urbit for you, consider a [hosting provider](#hosting-providers).
```

Comment:
The first line here should mention that this guide covers comets as well as planets.